README.md

```diff
@@ -43,6 +43,7 @@ This release includes 86 samples from 11 different cell lines.
 You can access the following data through the [AWS Open Data Registry](https://registry.opendata.aws/sgnex/):
 
 - raw files (fast5)
+- raw files (blow5)
```

Comment:
@cying111 Is it going to be under the same https://registry.opendata.aws/sgnex/ registry?